pylib/Utilities/ElkLogger.py

```diff
@@ -0,0 +1,132 @@
+
+from builtins import str
+#!/usr/bin/env python
+#
+# Copyright (c) 2015-2018 Intel, Inc. All rights reserved.
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+import sys
+import datetime
+from BaseMTTUtility import *
+import json
+import os
+from pathlib import Path
+import pwd
+import grp
+
+## @addtogroup Utilities
+# @{
+# @section ElkLogger
+# Log results to *.elog files with JSON entries to be later consumed by Elastic Stack (ELK)
+# Uses pymtt.py options: elk_head, elk_id, elk_testcycle, elk_testcase, elk_debug, elk_chown, elk_maxsize, elk_nostdout, elk_nostderr 
```

Comment:
can we explain what each of these are used for?

Reply:
elk_head, elk_id, elk_testcycle, elk_testcase, elk_debug, elk_chown, elk_maxsize, elk_nostdout, elk_nostderr  env vars that is